package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "clean": "rimraf ./dist",
     "gen": "npm run clean-gen && ./scripts/generate-client.sh",
-    "clean-gen": "rm -rf src/gen/* && rm -rf docs/openapi/*",
```

Comment:
Do we intend to keep `orchestration.swagger.json` in the case we don't remove the docs during each generation?

Reply:
oops missed the orchestration swagger somehow.. 

will close this PR and open a separate one